Docs/docs/platforms/android/setting-up-unity-for-creating-quest-content.md

```diff
@@ -6,14 +6,29 @@ createdAt: "2019-04-10T00:53:35.417Z"
 updatedAt: "2019-10-28T19:23:09.636Z"
 sidebar_position: -1
 ---
-Setting up Unity to create Quest content is actually pretty straightforward. This image covers it pretty well:
-![setting-up-unity-for-creating-quest-content-1ac8b19-VRChat_QuestContent_QuickStart.png](/img/setting-up-unity-for-creating-quest-content-1ac8b19-VRChat_QuestContent_QuickStart.png)
+![Building for Mobile instructions](/img/setting-up-unity-for-creating-quest-content-1ac8b19-VRChat_QuestContent_QuickStart.png)
 
-:::caution Requires Additional Setup
+If you're starting a brand new project, this won't take long at all. However, if you're converting a Windows platform project to an Android platform project, you will have to convert your assets appropriately. This can take quite a while for larger projects.
+
+For more details on best practices when working with dual-platform projects, read our documentation on [Cross-Platform Setup](/platforms/android/cross-platform-setup).
 
-[You need to install Unity's Android SDK](https://docs.unity3d.com/2019.4/Documentation/Manual/android-sdksetup.html). Otherwise the option won't pop up.
+## 3 Steps to get your world on VRChat for Quest
 
+### 1. Open Your Build Settings
+
+:::tip 
+Pros use the hotkeys Ctrl + Shift + B.
 :::
-If you're starting a brand new project, this won't take long at all. However, if you're converting a Windows platform project to an Android platform project, you will have to convert your assets appropriately. This can take quite a while for larger projects.
 
-For more details on best practices when working with dual-platform projects, read our documentation on [Cross-Platform Setup](/platforms/android/cross-platform-setup).
\ No newline at end of file
+### 2. Switch Platform to Android
+
+:::caution Requires Additional Setup
+If the Android option isn't appearing, [you'll need to install Unity's Android SDK.](https://docs.unity3d.com/Manual/android-sdksetup.html)
+
+:::
+
+Click "Android" -> "Switch Platform". Unity will automatically convert assets for Android-ready, it will take a while for larger projects with lots of assets.
+
```

Comment:
```suggestion
### 2. Switch Platform to Android

:::caution Requires Additional Setup
If the Android option isn't appearing, [you need to install Unity's Android SDK.](https://docs.unity3d.com/Manual/android-sdksetup.html)

:::

Click "Android" -> "Switch Platform". Unity will reimport your assets for Android. This takes a while for large projects with many assets.

```

Reply:
Added in https://github.com/vrchat-community/creator-docs/pull/166/commits/2b0d4b73b4355c39c4e7b9e9c6aea9df6709d511